rpcs3/Emu/GDB.cpp

```diff
@@ -12,40 +11,32 @@
 #include "Emu/Cell/RawSPUThread.h"
 #include "Emu/Cell/SPUThread.h"
 
-#ifndef _WIN32
-#include "fcntl.h"
+#ifdef _WIN32
+#include <winsock2.h>
+#include <WS2tcpip.h>
+#else
+#include <errno.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netinet/tcp.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <fcntl.h>
 #endif
 
 extern void ppu_set_breakpoint(u32 addr);
 extern void ppu_remove_breakpoint(u32 addr);
 
 LOG_CHANNEL(gdbDebugServer);
 
-int sock_init(void)
-{
-#ifdef _WIN32
-	WSADATA wsa_data;
-	return WSAStartup(MAKEWORD(1, 1), &wsa_data);
-#else
-	return 0;
-#endif
-}
-
-int sock_quit(void)
-{
-#ifdef _WIN32
-	return WSACleanup();
-#else
-	return 0;
-#endif
-}
-
 #ifndef _WIN32
-int closesocket(socket_t s) {
+int closesocket(int s)
+{
 	return close(s);
 }
```

Comment:
Does this function really need to exist anymore? Now that it's just an int, it's completely superfluous.

Reply:
It only exists in WinAPI...